plugins/ingest/build.gradle

```diff
@@ -28,6 +28,7 @@ dependencies {
     // we don't use Maxmind's http service:
     exclude group: 'com.google.http-client', module: 'google-http-client'
   }
+  compile 'joda-time:joda-time:2.8.2'
```

Comment:
I am not clear on whether this is needed or not. seems like everything works anyway but I wouldn't want ingest to depend on the forked joda that we have in es core.